packages/cli/package.json

```diff
@@ -65,7 +65,7 @@
     "typescript": "^4.0.3"
   },
   "oclif": {
-    "commands": "./src/commands",
+    "commands": "./dist/commands",
```

Comment:
I guess this makes sense, but are you sure and was this on purpose?

Reply:
Yes this was on purpose, earlier when I encountered the issue where it was still  `./lib/commands` I discussed it with @florian-richter and concluded that the path should be the path to the compiled folder not the source (`./disc` is the folder the `tsconfig` compiles to)